tools/ktx/ktx_main.cpp

```diff
@@ -100,7 +100,7 @@ class Tools : public Command {
 
 int Tools::main(int argc, char* argv[]) {
     cxxopts::Options options("ktx", "");
-    options.custom_help("[--version] [--help] <command> <command-args>");
+    options.custom_help("[<command>] [OPTION...]");
```

Comment:
Has the command really become optional, as implied by "[]"?

Is this message only printed for cases where there are no positional arguments?

Reply:
The command is optional, as you can call `ktx --version` and `ktx --help` alone (potentially with `--testrun` to output the default version.

This has been the case always, it's just that the custom help did not correctly reflect this, so I unified the syntax output to match the format used by the subcommands.